docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -136,3 +136,11 @@ Given I start mobile application with capabilities:
 |bstack:options.networkLogs     |true |
 |appium:options.clearSystemFiles|false|
 ----
+
+=== Verify elements state
+
+See xref:plugins:plugin-ui.adoc#_verify_elements_state[verify elements state]
+
+=== Verify elements number
+
+See xref:plugins:plugin-ui.adoc#_verify_elements_number[verify elements number]
```

Comment:
I think it makes sense to have a single reference here, there is not reason to enumerate all steps